optimum/intel/neural_compressor/quantization.py

```diff
@@ -265,6 +265,12 @@ def from_pretrained(
         else:
             model_class._keys_to_ignore_on_load_missing.extend(missing_keys_to_ignore_on_load)
 
+        if os.path.exists(os.path.join(model_name_or_path, "best_model.pt")):
```

Comment:
The `IncQuantizedModel` class should also allow the loading of models hosted on the Hugging Face Hub. Also our file name format is `pytorch_model.bin`